### PR TITLE
dot env template update

### PR DESCRIPTION
**Pull request by wojteknowacki**
Updated dot env template for mandatory fields to run e2e tests

### Pull Request Checklist

1. [ ] This code contains UI changes
2. [ ] All visible strings are translated with proper context including data-formatting
3. [ ] Attributes `data-test-id` are added for new elements
4. [ ] The changes are tested in Chrome/Firefox/Safari browsers and in light/dark mode
5. [ ] Your code works with the latest stable version of the core
6. [ ] I added changesets file (instructions in [contribution guide](https://github.com/saleor/saleor-dashboard/blob/main/.github/CONTRIBUTING.md)

### Test environment config

API_URI=https://automation-dashboard.staging.saleor.cloud/graphql/
APPS_MARKETPLACE_API_URI=https://apps.staging.saleor.io/api/v2/saleor-apps

### Do you want to run more stable tests?

To run all tests, just select the stable checkbox. To speed up tests, increase the number of containers. Tests will be re-run only when the "run e2e" label is added.

1. [ ] stable
2. [ ] app
3. [ ] attribute
4. [ ] category
5. [ ] collection
6. [ ] customer
7. [ ] giftCard
8. [ ] homePage
9. [ ] login
10. [ ] menuNavigation
11. [ ] navigation
12. [ ] orders
13. [ ] pages
14. [ ] payments
15. [ ] permissions
16. [ ] plugins
17. [ ] productType
18. [ ] products
19. [ ] sales
20. [ ] shipping
21. [ ] translations
22. [ ] variants
23. [ ] vouchers

CONTAINERS=2

### CODE REVIEW

File: .env.template [hunk 1]
```diff
@@ -1,4 +1,25 @@
...
+CYPRESS_PERMISSIONS_USERS_PASSWORD=xxxx
+CYPRESS_baseUrl=http://localhost:9000/
+
+
+E2E_USER_NAME=xxxx
+E2E_USER_PASSWORD=xxxx
+E2E_PERMISSIONS_USERS_PASSWORD=xxxx
+
+
+API_URI=https://demo.saleor.io/graphql/
```
Comment (Droniu) [inline .env.template:21]: **suggestion**: I would keep that at the top. This is the most important variable

File: .changeset/proud-garlics-count.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+"saleor-dashboard": minor
```
Comment (Droniu) [inline .changeset/proud-garlics-count.md:2]: **suggestion**: can be a patch I guess

File: .env.template [hunk 3]
```diff
@@ -1,4 +1,25 @@
...
+BASE_URL=http://localhost:9000/
+
+APPS_MARKETPLACE_API_URI=https://xxxx.staging.saleor.io/api/v2/saleor-apps
+CYPRESS_MAILPITURL=xxxx
```
Comment (poulch) [inline .env.template:25]: suggestion: this var should be placed with other cypess vars

File: .env.template [hunk 4]
```diff
@@ -1,4 +1,25 @@
...
+DEMO_MODE=false
+# for legacy reasons CYPRESS vars are needed
+CYPRESS_USER_NAME=xxxx
+CYPRESS_USER_PASSWORD=xxxx
+CYPRESS_SECOND_USER_NAME=xxxx
+CYPRESS_PERMISSIONS_USERS_PASSWORD=xxxx
+CYPRESS_baseUrl=http://localhost:9000/
+
+
+E2E_USER_NAME=xxxx
```
Comment (poulch) [inline .env.template:16]: suggestion: I'd give comment that this vars are used for Playwright

File: .env.template [hunk 5]
```diff
@@ -1,4 +1,25 @@
-API_URI=https://demo.saleor.io/graphql/
 APP_MOUNT_URI=/
 APPS_MARKETPLACE_API_URI=https://apps.saleor.io/api/v2/saleor-apps
-LOCALE_CODE="EN"
\ No newline at end of file
+LOCALE_CODE="EN" 
+
+
+# automation tests vars
+DEMO_MODE=false
```
Comment (poulch) [inline .env.template:7]: question: Is this var only part of automation tests, I don't think so

**Review by poulch — APPROVED** [review]

**Pull request merged**